boa_engine/src/string.rs

```diff
@@ -1,474 +1,559 @@
-    "setInt8",
-    "setInt16",
-    "setInt32",
-    "setUint8",
-    "setUint16",
-    "setUint32",
+    utf16!("DataView"),
+    utf16!("getBigInt64"),
+    utf16!("getBigUint64"),
+    utf16!("getFloat32"),
+    utf16!("getFloat64"),
+    utf16!("getInt8"),
+    utf16!("getInt16"),
+    utf16!("getInt32"),
+    utf16!("getUint8"),
+    utf16!("getUint16"),
+    utf16!("getUint32"),
+    utf16!("setBigInt64"),
+    utf16!("setBigUint64"),
+    utf16!("setFloat32"),
+    utf16!("setFloat64"),
+    utf16!("setInt8"),
+    utf16!("setInt16"),
+    utf16!("setInt32"),
+    utf16!("setUint8"),
+    utf16!("setUint16"),
+    utf16!("setUint32"),
     // Console object
-    "console",
-    "assert",
-    "debug",
-    "error",
-    "info",
-    "trace",
-    "warn",
-    "exception",
-    "count",
-    "countReset",
-    "group",
-    "groupCollapsed",
-    "groupEnd",
-    "time",
-    "timeLog",
-    "timeEnd",
-    "dir",
-    "dirxml",
+    utf16!("console"),
+    utf16!("assert"),
+    utf16!("debug"),
+    utf16!("error"),
+    utf16!("info"),
+    utf16!("trace"),
+    utf16!("warn"),
+    utf16!("exception"),
+    utf16!("count"),
+    utf16!("countReset"),
+    utf16!("group"),
+    utf16!("groupCollapsed"),
+    utf16!("groupEnd"),
+    utf16!("time"),
+    utf16!("timeLog"),
+    utf16!("timeEnd"),
+    utf16!("dir"),
+    utf16!("dirxml"),
     // Minified name
-    "a",
-    "b",
-    "c",
-    "d",
-    "e",
-    "f",
-    "g",
-    "h",
-    "i",
-    "j",
-    "k",
-    "l",
-    "m",
-    "n",
-    "o",
-    "p",
-    "q",
-    "r",
-    "s",
-    "t",
-    "u",
-    "v",
-    "w",
-    "x",
-    "y",
-    "z",
-    "A",
-    "B",
-    "C",
-    "D",
-    "E",
-    "F",
-    "G",
-    "H",
-    "I",
-    "J",
-    "K",
-    "L",
-    "M",
-    "N",
-    "O",
-    "P",
-    "Q",
-    "R",
-    "S",
-    "T",
-    "U",
-    "V",
-    "W",
-    "X",
-    "Y",
-    "Z",
-    "_",
-    "$",
+    utf16!("a"),
+    utf16!("b"),
+    utf16!("c"),
+    utf16!("d"),
+    utf16!("e"),
+    utf16!("f"),
+    utf16!("g"),
+    utf16!("h"),
+    utf16!("i"),
+    utf16!("j"),
+    utf16!("k"),
+    utf16!("l"),
+    utf16!("m"),
+    utf16!("n"),
+    utf16!("o"),
+    utf16!("p"),
+    utf16!("q"),
+    utf16!("r"),
+    utf16!("s"),
+    utf16!("t"),
+    utf16!("u"),
+    utf16!("v"),
+    utf16!("w"),
+    utf16!("x"),
+    utf16!("y"),
+    utf16!("z"),
+    utf16!("A"),
+    utf16!("B"),
+    utf16!("C"),
+    utf16!("D"),
+    utf16!("E"),
+    utf16!("F"),
+    utf16!("G"),
+    utf16!("H"),
+    utf16!("I"),
+    utf16!("J"),
+    utf16!("K"),
+    utf16!("L"),
+    utf16!("M"),
+    utf16!("N"),
+    utf16!("O"),
+    utf16!("P"),
+    utf16!("Q"),
+    utf16!("R"),
+    utf16!("S"),
+    utf16!("T"),
+    utf16!("U"),
+    utf16!("V"),
+    utf16!("W"),
+    utf16!("X"),
+    utf16!("Y"),
+    utf16!("Z"),
+    utf16!("_"),
+    utf16!("$"),
 ];
 
-const MAX_CONSTANT_STRING_LENGTH: usize = {
+const MAX_COMMON_STRING_LENGTH: usize = {
```

Comment:
I'm guessing this is the "maximum length of a string in the "common" list, right? Maybe this should be explicit in a documentation line.